CHANGELOG.md

```diff
@@ -9,10 +9,15 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.7.5](https://github.com/ASFHyP3/hyp3-autorift/compare/v0.7.4...v0.7.5)
 
-### Security
+### Added
```

Comment:
I think added is more in line with this change as "Security" is for vulnerabilities

![image](https://user-images.githubusercontent.com/7882693/152438218-800c03ba-8f6f-410f-8bfa-d2fd85045eed.png)